### ISSUE TITLE
ML-Benchmark prediction submission: ViT-IFCAv1-orog

### ISSUE THREAD

**Issue by jgonzalezab** [WCRP-CORDEX/ml-benchmark#42]
### Institution ID

IFCA

### Institution Name

Instituto de Física de Cantabria (IFCA), CSIC-Universidad de Cantabria

### Emulator identifier (`emulator_id`)

ViT-IFCAv1-orog

### Emulator Description

This emulator is based on the Vision Transformer (ViT) architecture [1]. The large-scale predictors are partitioned into patches, embedded, and passed through a series of Transformer blocks. The resulting patch representations are then upscaled to match the resolution of the local-scale predictand. As loss function it uses the CRPS-Spectral [2]. The noise is inyected in the ViT through conditional layer normalization [3]. Orography is added to the final embeddings of the Transformer block.

Input data are standardized at the grid-point level using the mean and standard deviation of the predictors corresponding to each training experiment. Models are trained for a variable number of epochs, determined via an early-stopping strategy based on a randomly split validation dataset.

### Hardware and Training Details

The model was trained on two V100 (32GB) GPUs using DDP. Each epoch takes approximately 30–40 seconds.

### Stochastic/Probabilistic Output

yes

### Reference URL

In preparation

### Repository for Reproducibility

https://github.com/jgonzalezab/ViT_CORDEX-ML-Bench
